### ISSUE TITLE
Save path not found

### ISSUE THREAD

**Issue by buvk** [shupershuff/D2rSPLoader#2]
```
Get-ChildItem : Cannot find path 'C:\Users\xxx\Saved Games\Diablo II
Resurrected\' because it does not exist.
At D:\Games\Diablo II\D2rSPLoader\D2rSPLoader.ps1:2110 char:19
+ ... haracters = Get-ChildItem -Path "$CharacterSavePath" -Filter "*.d2s"
...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\xxx\...II Resurre
   cted\:String) [Get-ChildItem], ItemNotFoundException
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.Ge
   tChildItemCommand
```

I previously moved my "Saved Games" folder to my OneDrive folder. This location is saved in Registry:
```
HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Shell Folders
{4C5C32FF-BB9D-43B0-B5B4-2D72E54EAAA4}
```

**Comment by shupershuff** [comment]
A good observation. Will ensure that the Saved Games folder is retrieved from registry instead of assuming it's in the default location

**Comment by shupershuff** [comment]
Resolved in v1.1.0

**Comment by shupershuff** [comment]
Resolved in v1.1.0
